custom_components/luxtronik/const.py

```diff
@@ -190,7 +190,7 @@ class LuxMode(Enum):
 
 
 # region Luxtronik Sensor ids
-LUX_SENSOR_DETECT_COOLING: Final = "calculations.ID_WEB_FreigabKuehl"
+LUX_SENSOR_DETECT_COOLING: Final = "parameters.ID_Sollwert_KuCft2_akt"
```

Comment:
This is the cooling setpoint for mixer circuit 2. That is only valid if the function of mixer circuit 2 is set to "Cool".
Are you sure this value is 0 when no cooling option is installed? Because for instance on my system, the setting ID_Sollwert_KuCft1_akt (cooling setpoint for mixer circuit 1) is 19.0, and I don't have mixer 1 in my setup.

Reply:
Thanks for the proposed detection method, I agree this should work more reliable. 
I updated the code to implement the proposed detection method
In a similar way I added detecting of the Cooling Target temperature sensor